source/exceptions.rst

```diff
@@ -135,6 +135,8 @@ arguments of an exception as part of the ``except`` statement::
 
 In Python 3, this is no longer true.
```

Comment:
This line should end with two colons, that way the `.. code-block:: python` line is not needed.